packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -791,12 +791,12 @@ Expected response body:
 }
 ```
 
-### Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_ManagedIdentityTrackedResources_createWithSystemAssigned
+### Azure_ResourceManager_CommonType_ManagedIdentity_Property_createWithSystemAssigned
```

Comment:
I think `CommonTypes` is better grammatically

Reply:
I also don't think we need a `_Property_` part of this name. I like just `Azure_Resource_Mangager_CommonTypes_ManagedIdentity_createWithSystemAssigned`

Reply:
`@armResourceOperations` have some autoRoute (and tag) related code within. And it is used by almost all ARM typespec spec.
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/src/resource.ts#L255-L256

Therefore, I'd wish we keep the operation group and the arm decorator, unless we are sure we don't want it in test.

Reply:
Thanks! I also prefer to have the interface. Adding `@autoRoute` is uncommon in current ARM practice.
Another option is that we could have the scenario name be `Azure_Resource_Mangager_CommonTypes_ManagedIdentity_ResourceProperty`, and have all the `create`, `get` and `update` grouped into the `ResourceProperty` interface.

Wonder if there is a way to define the full path of the scenario. 
e.g. `@scenario(fullName="Azure_Resource_Mangager_CommonTypes_ManagedIdentity_createWithSystemAssigned")`
That way, we could decouple scenario structure from the actual client structure.